python/src/aiconfig/editor/client/src/components/AIConfigEditor.tsx

```diff
@@ -148,6 +149,27 @@ export default function AIConfigEditor({
 
   const logEventHandler = callbacks?.logEventHandler;
 
+  const shareCallback = callbacks?.share;
+  const onShare = useCallback(async () => {
+    if (!shareCallback) {
+      return;
+    }
+    try {
+      // TODO: While uploading, show a loader state for share button
+      const { share_url: shareUrl } = await shareCallback();
+      // clientConfigToAIConfig(stateRef.current)
```

Comment:
We will want to do this if we want to share the config in `isDirty` state, rather than server source of truth. I think for simplicitly, better just not to include for now since Gradio will always have server updated